microhab/SPEXpdd.tex

```diff
@@ -0,0 +1,253 @@
+parallel) will be quite a bit more difficult; it requires a ton of datasheet
+reading, innovative code (not taking from examples), and customizing toolchains.
+
+PCB design accessibility is a lot more restricted. Members that have had PCB
+design classes (4th-5th year Electrical Engineers) or have experience from co-op
+will be able to contribute to the design.
+
+
+
+\section{Implementation}
+\label{sec:implementation}
+  % What path do you anticipate the project to take?
+Documentation for each step of the respected engineering paths is required. It
+is important during the implementation aspect to retain problems experienced and
+the solutions used to overcome.
+
+\subsection{Deliverables}
+\label{subsec:deliverables}
+  % When all is said and done, what will you have to show for it?
+  % Examples: Hardware, software, poster, ImagineRIT demo, presentations, technical papers...
+\begin{itemize}
+    \item Bill of Materials for Hardware
+    \item PCB schematics
+    \item Arduino software source files
+    \item C software source files
+    \item Fully assembled balloon, parachute, and payload
+\end{itemize}
+
+\subsection{Milestones}
+\label{subsec:milestones}
+  % Be as detailed as you can, but it's okay if there are unknowns.
+  % At the very least, specify how many semester you expect the project to take until it reaches completion.
+\subsubsection*{Electrical}
+\begin{itemize}
+    \item 2 weeks --- PCB design
+    \item 1 week --- PCB reviews
+    \item 1 week --- Order hardware
+    \item 1 week --- PCB assembly
+    \item 1 week --- PCB testing
+\end{itemize}
+
+\subsubsection*{Arduino software}
+\begin{itemize}
+    \item 1 week --- gather libraries needed
+    \item 1 week --- plan out software design and flow
+    \item 2 weeks --- write software
+    \item 1 week --- test software
+    \item 1 week --- neaten up software
+\end{itemize}
+
+\subsubsection*{C software}
+\begin{itemize}
+    \item 1 week --- prepare standard development environment
+    \item 1 week --- plan out software design and flow
+    \item 4 weeks --- create drivers
+    \item 8 weeks --- write software
+    \item 2 weeks --- test software
+    \item 1 week --- neaten up software
+\end{itemize}
+\section{Externalities}
+  % Things not directly related to the work or outcomes, but related to the project as a whole.
+\subsection{Prerequisite Skills}
+  % Which skills do team members need to have before work can start (not including skills that will be learned ``on the job'')?
+
+  % Estimate costs that would be needed to meet objectives.
+Covered in Accessibility.
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+\textmu{}Hab should be able to be completely engineered by students. Faculty
```

Comment:
I think that faculty members could be involved in the design of things.
Projects engineered COMPLETELY by students (no faculty help) are much more prone to error, stupid mistakes, more clunky...

As an alum, I can say with full confidence that you should use faculty help as much as you can.

I don't mean cc'ing them on every email or requiring them to be in design reviews, but at least approach them with questions where you feel uncertain and things like that.